test/e2e/specs/specs-wpcom/wp-gutenberg__experimental.ts

```diff
@@ -0,0 +1,86 @@
+/**
+ * @group gutenberg
+ */
+
+import {
+	setupHooks,
+	TestAccount,
+	BrowserHelper,
+	DataHelper,
+	GutenbergEditorPage,
+} from '@automattic/calypso-e2e';
+import { Page } from 'playwright';
+
+describe( DataHelper.createSuiteTitle( 'Gutenberg: Experimental Features' ), function () {
+	const accountName = BrowserHelper.targetGutenbergEdge()
+		? 'gutenbergSimpleSiteEdgeUser'
+		: 'gutenbergSimpleSiteUser';
+
+	let page: Page;
+	let gutenbergEditorPage: GutenbergEditorPage;
+
+	setupHooks( async ( args ) => {
+		page = args.page;
+
+		const testAccount = new TestAccount( accountName );
+		await testAccount.authenticate( page );
+	} );
+
+	it( 'Go to the new post page', async function () {
+		gutenbergEditorPage = new GutenbergEditorPage( page );
+		await gutenbergEditorPage.visit( 'post' );
+	} );
+
+	it.each( [
+		[ 'blockEditor', '__unstableInserterMenuExtension', 'function' ],
+		[ 'date', '__experimentalGetSettings', 'function' ],
+		[ 'components', '__experimentalNavigationBackButton', 'object' ],
+		[ 'editPost', '__experimentalMainDashboardButton', 'function' ],
```

Comment:
This modified list of experimental features is used over the object in the Selenium spec, which is:

```
const EXPERIMENTAL_FEATURES = {
	'@wordpress/block-editor': [ '__unstableInserterMenuExtension' ],
	'@wordpress/date': [ '__experimentalGetSettings' ],
	'@wordpress/components': [ '__experimentalNavigationBackButton' ],
	'@wordpress/edit-post': [ '__experimentalMainDashboardButton' ],
};
```

In the Selenium spec, operations are performed on the object key to remove the `@wordpress/` prefix. Additionally hyphens are removed and the resulting package name turned into camelCase. 

These operations appear superfluous and not related to the actual testing of experimental features. Therefore, I've made the decision to define the package name as it is expected to be returned.